src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld.java

```diff
@@ -169,6 +170,11 @@ public boolean testForCubes(CubePos start, CubePos end, Predicate<Cube> cubeAllo
         return this.chunkProvider.provideChunk(blockToCube(blockX), blockToCube(blockZ))
                 .getHeightValue(blockToLocal(blockX), blockToLocal(blockZ));
     }
+    
+    @Override public int updateLCG() {
```

Comment:
Isn't there a vanilla method that does the same thing? You could use `@Shadow` is there is. If there isn't, you are adding unnecessary changes to `updateBlocks`